.resinci.yml

```diff
@@ -0,0 +1 @@
+disabled: true
```

Comment:
You can remove this file altogether since balenaCI will be auto-disabled if it detects flowzone (it looks for a `.github/workflows/flowzone.yml` file)
```suggestion
```